sda-commons-server-openapi-example/src/main/java/org/sdase/commons/server/openapi/example/people/rest/Api.java

```diff
@@ -0,0 +1,56 @@
+package org.sdase.commons.server.openapi.example.people.rest;
+
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import io.swagger.v3.oas.annotations.enums.SecuritySchemeIn;
+import io.swagger.v3.oas.annotations.enums.SecuritySchemeType;
+import io.swagger.v3.oas.annotations.info.Contact;
+import io.swagger.v3.oas.annotations.info.Info;
+import io.swagger.v3.oas.annotations.info.License;
+import io.swagger.v3.oas.annotations.security.SecurityScheme;
+import io.swagger.v3.oas.annotations.tags.Tag;
+
+// Your application might have an additional place were global swagger annotations are defined, like
+// the App interface. It's only purpose is do define the annotations in the right scope.
+@OpenAPIDefinition(
+    info =
+        @Info(
+            // Set the name of the service as title of the Swagger documentation.
+            title = "Swagger Example Application",
+            // Set the description of the API, like an introduction. Like in most of the fields you
+            // may use markdown here to apply custom formatting.
+            description =
+                "This is the API documentation for the **People API**.\n\n"
+                    + "The API provides operations for managing and searching people.",
+            // Set the version of the API, which is a string. By default version 1 is used.
+            version = "3",
+            // Set the contact information of the API author.
+            contact =
+                @Contact(
+                    name = "SDA SE",
+                    email = "info@example.com",
+                    url = "https://myfuture.sda-se.com"),
+
+            // Set the name of the license of the API as well as an url to the full license
+            // information. Note the license is only an example.
+            license = @License(name = "MIT", url = "https://opensource.org/licenses/MIT"),
+            // For APIs with public availability, terms of service might be required.
+            // Set the url to the terms of service here.
+            termsOfService = "https://www.sda-se.com/legal-notice/"),
+    tags = {@Tag(name = "People", description = "API features related to *People*")})
+// Define the security definitions that are later referenced inside the api definitions.
+// Here two are defined, one for the consumer token and the access token required for every
+// call.
+@SecurityScheme(
+    type = SecuritySchemeType.APIKEY,
+    description = "Passes the Consumer-Token to the service class.",
+    name = AuthDefinition.CONSUMER_TOKEN,
+    paramName = "Consumer-Token",
+    in = SecuritySchemeIn.HEADER)
+@SecurityScheme(
+    type = SecuritySchemeType.HTTP,
```

Comment:
Is there a reason to use `HTTP` instead of `OPENIDCONNECT`?  Should that reason be documented in the example?

Reply:
Btw: OpenID Connect is not supported by the SwaggerUI (see https://swagger.io/docs/specification/authentication/openid-connect-discovery/). But the service owners are free to explore the OpenAPI 3.0 specification and configure it according to their needs.